.buildkite/pipeline.yml

```diff
@@ -380,10 +380,26 @@ steps:
           slurm_gpus: 1
 
       # GPU RUNS: AMIP
-      - label: "GPU AMIP target: albedo from function"
-        key: "gpu_target_amip_albedo_function"
-        command: "julia --color=yes --project=experiments/AMIP/ experiments/AMIP/coupler_driver.jl --config_file $CONFIG_PATH/gpu_target_amip_albedo_function.yml"
-        artifact_paths: "experiments/AMIP/output/amip/gpu_target_amip_albedo_function_artifacts/*"
+      - label: "GPU AMIP test: albedo from function"
+        key: "gpu_amip_albedo_function"
```

Comment:
I just renamed this config file/key to avoid confusion with the target GPU AMIP runs added in this PR

Reply:
Sounds good! Target usually alludes to the target resolution (i.e. 16 h_elem). Could we use the low resolution here, i.e. 6 h_elem (also correcting for dt and vertical resolution - the `coarse_single_ft64_hourly_checkpoints` could be used as a reference here).